retrieval/k8s/k8s_stats.go

```diff
@@ -239,6 +291,12 @@ func sanitizePod(to interface{}) interface{} {
 	for j, container := range (*cast).Spec.InitContainers {
 		(*cast).Spec.InitContainers[j] = sanitizeContainer(container)
 	}
+	(*cast).Spec.DNSPolicy = ""
```

Comment:
are we able to completely remove the field from the struct or can we only set it to empty?

Reply:
Since it's underlying type is a string, setting it to "" should drop it because the json has a `omitempty` tag.
It is how we sanitize all primitive data types